index.html

```diff
@@ -35,6 +35,236 @@ <h3>LET'S CONNECT</h3>
+        </div>
+        
+        <div class="left-block">
+          <div class="primary-text">
+            <h2 class="project_title">Multi-Post Stories</h2>
+            <ul>
+              <li class="client">CANOPY</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">Back End Dev</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">2015</li>
+            </ul>
+          </div>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+
+          <div class="tags">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+
+          <div class="project-link">
+            <div class="project-view">
+              <h3>See Project</h3>
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="works-1">
+        <div class="snapshot">
+          <img src="images/snapshoot-3.png" alt="snapshot">
+        </div>
+        
+        <div class="left-block">
+          <div class="primary-text">
+            <h2 class="project_title">Tonic</h2>
+            <ul>
+              <li class="client">CANOPY</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">Back End Dev</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">2015</li>
+            </ul>
+          </div>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+
+          <div class="tags">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+
+          <div class="project-link">
+            <div class="project-view">
+              <h3>See Project</h3>
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="works-1">
+        <div class="snapshot">
+          <img src="images/snapshoot-4.png" alt="snapshot">
+        </div>
+        
+        <div class="left-block">
+          <div class="primary-text">
+            <h2 class="project_title">Multi-Post Stories</h2>
+            <ul>
+              <li class="client">CANOPY</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">Back End Dev</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">2015</li>
+            </ul>
+          </div>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+
+          <div class="tags">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
```

Comment:
reference | your copy
------------ | -------------
![image](https://user-images.githubusercontent.com/67957533/129378243-f580b4da-7df8-41d9-a057-b03ab399c431.png) | ![image](https://user-images.githubusercontent.com/67957533/129378202-07f1d78f-9024-4c88-bc48-918a53f0610c.png)

- [ ] I suggest you use a `width: min-content`  so you don't have overflow from the text. looks like you are using a fixed width 🤷‍♂️